localization/languages/ko.json

```diff
@@ -13,6 +13,14 @@
         "saveImageAs": "이미지를 다른 이름으로 저장",
         "searchWith": "%s에서 검색", //%s will be replaced with the name of the current search engine
         "copyLink": "주소 복사",
+        "select-all": "Select All", // Missing Translation
+        "undo": "Undo", // Missing Translation
+        "redo": "Redo", // Missing Translation
+        "cut": "Cut", // Missing Translation
+        "select-all": "Select All", // Missing Translation
```

Comment:
It looks like these maybe got duplicated, can you remove the second set of strings?